torchvision/prototype/datasets/_builtin/food101.py

```diff
@@ -0,0 +1,86 @@
+import functools
+from pathlib import Path
+from typing import Any, Tuple, List, Dict, cast, Optional, BinaryIO
+
+from torchdata.datapipes.iter import IterDataPipe, Filter, Mapper, CSVParser, JsonParser, Demultiplexer
+from torchvision.prototype.datasets.utils import Dataset, DatasetInfo, DatasetConfig, HttpResource, OnlineResource
+from torchvision.prototype.datasets.utils._internal import (
+    hint_shuffling,
+    hint_sharding,
+    path_comparator,
+    getitem,
+    INFINITE_BUFFER_SIZE,
+)
+from torchvision.prototype.features import Label, EncodedImage
+
+
+class Food101(Dataset):
+    def _make_info(self) -> DatasetInfo:
+        return DatasetInfo(
+            "food101",
+            homepage="https://data.vision.ee.ethz.ch/cvl/datasets_extra/food-101",
+            # Only train and test splits, no validation.
```

Comment:
That should be obvious from only specifying the two. Datasets are weird. No one is going to bat an eye for "non-standard" option :wink:

Reply:
haha, indeed, will remove it. :+1: